Jenkinsfile.integration

```diff
@@ -26,6 +26,7 @@ pipeline {
         choice(choices: ['caasp3', 'caasp4'], description: 'Which caasp version to use?', name: 'caasp_version')
         booleanParam(name: 'run_tempest', defaultValue: false, description: 'Run tempest tests?')
         choice(choices: ['smoke', 'all'], description: 'Run smoke or all tempest tests?', name: 'tempest_test_type')
+        choice(choices: ['ses_aio', 'ses_rook'], description: 'Which ses version to use?', name: 'ses_version')
```

Comment:
Just for my understanding - does `ses_aio` mean SES5 and `ses_rook` SES6?

Reply:
yep, ses_aio was the default name because of the all-in-one approach to having just one node. ses_rook could be 6 or 7 I think, not clean on that one yet.